helm-util/helm.libsonnet

```diff
@@ -0,0 +1,57 @@
+local d = import 'github.com/sh0rez/docsonnet/doc-util';
+{
+  local this = self,
+
+  '#': d.pkg(
+    name='helm-util',
+    url='github.com/grafana/jsonnet-libs/helm-util/helm.libsonnet',
+    help='`helm-util` provides utilities for using helm in jsonnet',
+  ),
+
+  '#helmTemplate': d.fn(
+    '`helmTemplate` wraps the helmTemplate native function and returns a rendered chart as a map of kubernetes manifests',
+    [
+      d.arg('name', d.T.string),
+      d.arg('chart', d.T.string),
+      d.arg('conf', d.T.object),
+      d.arg('labels', d.T.object),
+    ]
+  ),
+  helmTemplate(name, chart, conf={}, labels={})::
```

Comment:
This function signature is already rather long. Given you already have an opts-like object (conf) in here, I think labels should be part of that instead.

Generally spoken once you have a lot of function parameters, I prefer to only have these that are **required** as parameters, and those that are optional in an final object parameter

Reply:
I'd rather drop the `labels` parameter completely than put it into `conf`, these arguments have nothing to do with each other in lower level code.

Reply:
Maybe we can have a `withLabels` function on here?

Reply:
Right, then let's remove labels entirely from `helmTemplate` and have it as a separate func